.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+* viper Version: x.x.x
+* pyethereum Version: x.x.x
+* OS: osx/linux/win
+* Python Version (python --version):
+* Environment (output of `pip freeze`):
+
+### What is wrong?
```

Comment:
May be a issue could not be a trouble, what do you think?

Reply:
I see your point, what do you think of `What's your issue about?`

Reply:
Yes, it's better